[0m10:06:32.506430 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022C85B55B40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022C88658E50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022C8865A650>]}


============================== 10:06:32.506430 | 809e8e2b-0c73-4789-821b-7a14376708fa ==============================
[0m10:06:32.506430 [info ] [MainThread]: Running with dbt=1.7.19
[0m10:06:32.506430 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\DELL\\.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m10:06:32.522092 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': WindowsPath('C:/Users/DELL/.dbt')}
[0m10:06:32.522092 [info ] [MainThread]: Creating dbt configuration folder at 
[0m10:07:02.964911 [debug] [MainThread]: Starter project path: C:\Code\DBT\DBTProject-mfgdatabase\DBTProject-manufacturingdb\venv\lib\site-packages\dbt\include\starter_project
[0m10:07:03.048919 [info ] [MainThread]: 
Your new dbt project "dbt_sf_mfgdb" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m10:07:03.051971 [info ] [MainThread]: Setting up your profile.
[0m10:10:10.148572 [info ] [MainThread]: Profile dbt_sf_mfgdb written to C:\Users\DELL\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m10:10:10.148572 [debug] [MainThread]: Command `dbt init` succeeded at 10:10:10.148572 after 217.72 seconds
[0m10:10:10.148572 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022C85B55B40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022C886582B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022C886AB250>]}
[0m10:10:10.148572 [debug] [MainThread]: Flushing usage events
[0m10:10:26.995646 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A7C89C53F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A7CB4CA860>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A7CB4CA5F0>]}


============================== 10:10:26.995646 | aad84d62-0d53-4dd5-b3fd-57f87d027247 ==============================
[0m10:10:26.995646 [info ] [MainThread]: Running with dbt=1.7.19
[0m10:10:26.995646 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': 'C:\\Users\\DELL\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m10:10:26.995646 [info ] [MainThread]: dbt version: 1.7.19
[0m10:10:26.995646 [info ] [MainThread]: python version: 3.10.11
[0m10:10:26.995646 [info ] [MainThread]: python path: C:\Code\DBT\DBTProject-mfgdatabase\DBTProject-manufacturingdb\venv\Scripts\python.exe
[0m10:10:26.995646 [info ] [MainThread]: os info: Windows-10-10.0.19045-SP0
[0m10:10:27.324291 [info ] [MainThread]: Using profiles dir at C:\Users\DELL\.dbt
[0m10:10:27.324291 [info ] [MainThread]: Using profiles.yml file at C:\Users\DELL\.dbt\profiles.yml
[0m10:10:27.324291 [info ] [MainThread]: Using dbt_project.yml file at C:\Code\DBT\DBTProject-mfgdatabase\DBTProject-manufacturingdb\dbt_project.yml
[0m10:10:27.339917 [info ] [MainThread]: adapter type: snowflake
[0m10:10:27.339917 [info ] [MainThread]: adapter version: 1.7.1
[0m10:10:27.339917 [info ] [MainThread]: Configuration:
[0m10:10:27.339917 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m10:10:27.343733 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m10:10:27.345162 [info ] [MainThread]: Required dependencies:
[0m10:10:27.346148 [debug] [MainThread]: Executing "git --help"
[0m10:10:27.391734 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--no-lazy-fetch]\n           [--no-optional-locks] [--no-advice] [--bare] [--git-dir=<path>]\n           [--work-tree=<path>] [--namespace=<name>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone      Clone a repository into a new directory\n   init       Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add        Add file contents to the index\n   mv         Move or rename a file, a directory, or a symlink\n   restore    Restore working tree files\n   rm         Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect     Use binary search to find the commit that introduced a bug\n   diff       Show changes between commits, commit and working tree, etc\n   grep       Print lines matching a pattern\n   log        Show commit logs\n   show       Show various types of objects\n   status     Show the working tree status\n\ngrow, mark and tweak your common history\n   backfill   Download missing objects in a partial clone\n   branch     List, create, or delete branches\n   commit     Record changes to the repository\n   merge      Join two or more development histories together\n   rebase     Reapply commits on top of another base tip\n   reset      Reset current HEAD to the specified state\n   switch     Switch branches\n   tag        Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch      Download objects and refs from another repository\n   pull       Fetch from and integrate with another repository or a local branch\n   push       Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m10:10:27.391734 [debug] [MainThread]: STDERR: "b''"
[0m10:10:27.391734 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m10:10:27.391734 [info ] [MainThread]: Connection:
[0m10:10:27.391734 [info ] [MainThread]:   account: kitxcef-yi17147
[0m10:10:27.391734 [info ] [MainThread]:   user: HIMANSHU
[0m10:10:27.391734 [info ] [MainThread]:   database: VISK_INDUS_PVT_LTD
[0m10:10:27.405280 [info ] [MainThread]:   warehouse: COMPUTE_WH
[0m10:10:27.406278 [info ] [MainThread]:   role: ACCOUNTADMIN
[0m10:10:27.407278 [info ] [MainThread]:   schema: PUBLIC
[0m10:10:27.409278 [info ] [MainThread]:   authenticator: None
[0m10:10:27.410283 [info ] [MainThread]:   private_key_path: None
[0m10:10:27.410448 [info ] [MainThread]:   token: None
[0m10:10:27.410448 [info ] [MainThread]:   oauth_client_id: None
[0m10:10:27.410448 [info ] [MainThread]:   query_tag: None
[0m10:10:27.410448 [info ] [MainThread]:   client_session_keep_alive: False
[0m10:10:27.410448 [info ] [MainThread]:   host: None
[0m10:10:27.410448 [info ] [MainThread]:   port: None
[0m10:10:27.410448 [info ] [MainThread]:   proxy_host: None
[0m10:10:27.410448 [info ] [MainThread]:   proxy_port: None
[0m10:10:27.410448 [info ] [MainThread]:   protocol: None
[0m10:10:27.410448 [info ] [MainThread]:   connect_retries: 1
[0m10:10:27.410448 [info ] [MainThread]:   connect_timeout: None
[0m10:10:27.410448 [info ] [MainThread]:   retry_on_database_errors: False
[0m10:10:27.410448 [info ] [MainThread]:   retry_all: False
[0m10:10:27.410448 [info ] [MainThread]:   insecure_mode: False
[0m10:10:27.421883 [info ] [MainThread]:   reuse_connections: None
[0m10:10:27.422895 [info ] [MainThread]: Registered adapter: snowflake=1.7.1
[0m10:10:27.424634 [debug] [MainThread]: Acquiring new snowflake connection 'debug'
[0m10:10:27.424634 [debug] [MainThread]: Using snowflake connection "debug"
[0m10:10:27.424634 [debug] [MainThread]: On debug: select 1 as id
[0m10:10:27.424634 [debug] [MainThread]: Opening a new connection, currently in state init
[0m10:10:28.249660 [debug] [MainThread]: Snowflake adapter: Error running SQL: select 1 as id
[0m10:10:28.249660 [debug] [MainThread]: Snowflake adapter: Rolling back transaction.
[0m10:10:28.249660 [debug] [MainThread]: On debug: No close available on handle
[0m10:10:28.249660 [info ] [MainThread]:   Connection test: [[31mERROR[0m]

[0m10:10:28.249660 [info ] [MainThread]: [31m2 checks failed:[0m
[0m10:10:28.249660 [info ] [MainThread]: Project loading failed for the following reason:
 project path <C:\Code\DBT\DBTProject-mfgdatabase\DBTProject-manufacturingdb\dbt_project.yml> not found

[0m10:10:28.249660 [info ] [MainThread]: dbt was unable to connect to the specified database.
The database returned the following error:

  >Database Error
  250001 (08001): Failed to connect to DB: kitxcef-yi17147.snowflakecomputing.com:443. Incorrect username or password was specified.

Check your database credentials and try again. For more information, visit:
https://docs.getdbt.com/docs/configure-your-profile


[0m10:10:28.249660 [debug] [MainThread]: Command `dbt debug` failed at 10:10:28.249660 after 1.34 seconds
[0m10:10:28.249660 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m10:10:28.249660 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A7C89C53F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A7CB4C93F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A7CBC134C0>]}
[0m10:10:28.265274 [debug] [MainThread]: Flushing usage events
[0m17:58:51.123777 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E15F479B40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E161F74040>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E161F77490>]}


============================== 17:58:51.140547 | f112ba4e-30f1-487f-a235-004668cdc766 ==============================
[0m17:58:51.140547 [info ] [MainThread]: Running with dbt=1.7.19
[0m17:58:51.146832 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\DELL\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt docs generate', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m17:58:51.148872 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Code\DBT\DBTProject-mfgdatabase\DBTProject-manufacturingdb\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m17:58:51.154249 [debug] [MainThread]: Command `dbt docs generate` failed at 17:58:51.154249 after 0.15 seconds
[0m17:58:51.156445 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E15F479B40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E161F75B10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E161F77430>]}
[0m17:58:51.158512 [debug] [MainThread]: Flushing usage events
[0m18:05:16.319234 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F9F8689B40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F9FB18DE40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F9FB18F160>]}


============================== 18:05:16.320292 | 8f4e16f5-b93c-4b02-99b3-cc06a897db2c ==============================
[0m18:05:16.320292 [info ] [MainThread]: Running with dbt=1.7.19
[0m18:05:16.329614 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': 'C:\\Users\\DELL\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt run -s gold.*', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m18:05:16.330682 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Code\DBT\DBTProject-mfgdatabase\DBTProject-manufacturingdb\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m18:05:16.335454 [debug] [MainThread]: Command `dbt run` failed at 18:05:16.335454 after 0.14 seconds
[0m18:05:16.337938 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F9F8689B40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F9FB18E380>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F9FB18E560>]}
[0m18:05:16.340725 [debug] [MainThread]: Flushing usage events
